source/GameAction.cpp

```diff
@@ -190,7 +193,17 @@ void GameAction::LoadSingle(const DataNode &child)
 			swap(minDays, maxDays);
 		events[GameData::Events().Get(child.Token(1))] = make_pair(minDays, maxDays);
 	}
-	else if(key == "fail" && child.Size() >= 2)
+	else if(key == "music" && hasValue && !child.Token(1).empty())
+	{
+		music = child.Token(1);
+		playMusic = true;
+	}
+	else if(key == "quiet")
+	{
+		music = "";
+		playMusic = true;
```

Comment:
We could turn `music` into a std::optional, saving the need for the extra boolean variable.

Reply:
Done